.buildkite/scripts/steps/cloud/purge_projects.ts

```diff
@@ -10,7 +10,7 @@ import { execSync } from 'child_process';
 import axios from 'axios';
 
 async function getPrProjects() {
-  const match = /^kibana-pr-([0-9]+)-(elasticsearch|security|observability)$/;
+  const match = /^(keep.?)?kibana-pr-([0-9]+)-(elasticsearch|security|observability)$/;
```

Comment:
@jbudz I've updated this regex, but don't know of a good way to test it since that job runs on a scheduled basis in Buildkite against the `main` branch.

Below is a set of successful regex tests for this pattern (note those that "Match full" vs "Match none")
<img width="1310" alt="Screenshot 2024-02-09 at 12 41 13 PM" src="https://github.com/elastic/kibana/assets/147995946/0a017397-f622-42d0-bbb7-21e52226a4b4">

Let me know if there's anything else you'd like done to test this purge regex.

Reply:
Thank you!  LGTM.  I'll verify after it's merged and update if necessary